Configure ODF for NESE storage

larsks opened the pull request:
Configure ODF to use NESE as an external storage cluster.

larsks committed:
Fake metrics endpoint to make ODF happy

While the "monitoring-endpoint" configuration in OCS 4.8 was optional,
it appears to be required in 4.10, and ODF refuses to deploy if it
can't reach the endpoint.

This commit provides a fake metrics endpoint to make the ODF operator
happy.

larsks committed:
Configure ODF for NESE storage

Configure ODF to use NESE as an external storage cluster.

mikthoma reviewed (approved):
Lot of work here. LGTM..

larsks committed:
Configure OpenShift console

Ensure that all console plugins are enabled. Add NERC custom logo.

x-branch: feature/odf

computate reviewed (approved):
Looks good, I'm learning more about your Kustomizations.
